components/OptimusPage/OptimusHero.jsx

```diff
@@ -0,0 +1,41 @@
+import Image from 'next/image';
+
+import { MAIN_TITLE_CLASS, TEXT_MEDIUM_LIGHT_CLASS } from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+
+const LeftContent = () => (
+  <div className="lg:col-span-5 lg:col-start-2 lg:p-0 lg:text-left lg:gap-0 lg:items-start md:mb-12 flex-col flex text-center item-center px-5 mb-2 gap-6">
+    <div
+      className={`${TEXT_MEDIUM_LIGHT_CLASS} lg:self-start mb-2 self-center`}
+    >
+      OLAS OPTIMUS
+    </div>
+
+    <h2 className={MAIN_TITLE_CLASS}>
+      AI Agent-powered
```

Comment:
tiny
```suggestion
      AI agent-powered
```

Reply:
I think the capital A works better as the other words are capitalised in the Figma?

Reply:
yeah.. i see what you mean 🤔 

ideally this would be "AI agent-powered DeFi management", as it kinda reads like "AI Agent" is our product i.e. "Diet Coke". "Defi Management" also reads like a Thing too, but powered by a product/service called "AI Agent". 

But, semantics, doesn't matter. Looks good either way. Approved!

Reply:
I think we did end up changing it to "AI agent-powered DeFi management" so works out quite nicely!